.github/workflows/sync_typeshed.yaml

```diff
@@ -36,9 +36,21 @@ jobs:
         run: |
           git config --global user.name typeshedbot
           git config --global user.email '<>'
+      - uses: astral-sh/setup-uv@bd01e18f51369d5a26f1651c3cb451d3417e3bba # v6.3.1
       - name: Sync typeshed
         id: sync
         run: |
+          # Run with the full matrix of Python versions supported by typeshed,
+          # so that we codemod in docstrings that only exist on certain versions.
+          # TODO: what about platform-specific APIs?
```

Comment:
I think we should create an issue for this instead of this ominous TODO comment where it isn't directly clear what hte issue is with platform specific APIs

Reply:
The issue is that we get the runtime docstrings by inspecting the docstrings at runtime, so if an API doesn't exist at runtime (because e.g. it's Windows-specific and we're running on Linux), then we won't add a docstring to it.

This is probably solvable, but it might require complicating the GitHub workflow a bit.